homeassistant/__main__.py

```diff
@@ -152,6 +161,42 @@ def write_pid(pid_file):
         sys.exit(1)
 
 
+def install_osx():
+    """ Setup to run via launchd on OS X """
+    hass_path = os.popen('which hass').read().strip()
```

Comment:
Need `with` blocks for these commands too or else the file objects will not be closed.